File: src/main/java/no/nav/melosys/eessi/models/sed/nav/VedtakA010.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

@SuppressWarnings("unused")
@JsonInclude(JsonInclude.Include.NON_NULL)
@Data
public class VedtakA010 {

    private String gjeldervarighetyrkesaktivitet;

    private String eropprinneligvedtak;

    private PeriodeA010 gjelderperiode;

    private String land;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/VedtakA010.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class VedtakA010(
    var gjeldervarighetyrkesaktivitet: String? = null,
    var gjelderperiode: PeriodeA010? = null,
    var land: String? = null,
) : Vedtak()
